go-indexer/indexer/indexer.go

```
@@ -172,202 +163,225 @@ func (i *Indexer) Start() error {
+					}
+
+					err := i.subscribeToEvent(contractAddress, event)
+
+					if err != nil {
+						// Use non-blocking send to avoid panic if channel is closed
+						select {
+						case errors <- fmt.Errorf("failed to subscribe to event %s for contract %s: %w", event.Name, contract.Address, err):
+						default:
+							// Channel is closed or full, ignore
+						}
+						i.Stop()
```

golangci-lint (1.64.8)

Error return value of `i.Stop` is not checked

(errcheck)